contrib/setup-win32.nsi

```diff
@@ -0,0 +1,135 @@
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libunistring-2.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libusb-1.0.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libwinpthread-1.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/libxml2-2.dll"
+  File "/usr/x86_64-w64-mingw32/sys-root/mingw/bin/zlib1.dll"
+
+  # fwupd
+  File "dfu-tool.exe"
+  File "fwupd/fwupdagent.exe"
+  File "fwupd/fwupdtool.exe"
+  File "gcab.exe"
+  File "libfwupd-2.dll"
+  File "libfwupdplugin-1.dll"
+  File "libgcab-1.0-0.dll"
+  File "libxmlb-1.dll"
+  SetOutPath "$INSTDIR\fwupd-plugins-3"
+  File "fwupd-plugins-3/libfu_plugin_colorhug.dll"
+  File "fwupd-plugins-3/libfu_plugin_csr.dll"
+  File "fwupd-plugins-3/libfu_plugin_dell_dock.dll"
+  File "fwupd-plugins-3/libfu_plugin_dfu.dll"
+  File "fwupd-plugins-3/libfu_plugin_ebitdo.dll"
+  File "fwupd-plugins-3/libfu_plugin_fastboot.dll"
+  File "fwupd-plugins-3/libfu_plugin_jabra.dll"
+  File "fwupd-plugins-3/libfu_plugin_nitrokey.dll"
+  File "fwupd-plugins-3/libfu_plugin_rts54hid.dll"
+  File "fwupd-plugins-3/libfu_plugin_rts54hub.dll"
+  File "fwupd-plugins-3/libfu_plugin_solokey.dll"
+  File "fwupd-plugins-3/libfu_plugin_steelseries.dll"
+  File "fwupd-plugins-3/libfu_plugin_synaptics_cxaudio.dll"
+  File "fwupd-plugins-3/libfu_plugin_synaptics_prometheus.dll"
+  File "fwupd-plugins-3/libfu_plugin_upower.dll"
+  File "fwupd-plugins-3/libfu_plugin_vli.dll"
+  File "fwupd-plugins-3/libfu_plugin_wacom_usb.dll"
+  SetOutPath "$INSTDIR\etc\pki\fwupd"
+  File "etc/pki/fwupd/LVFS-CA.pem"
+  File "etc/pki/fwupd-metadata/LVFS-CA.pem"
+  SetOutPath "$INSTDIR\share\fwupd\quirks.d"
+  File "share/fwupd/quirks.d/colorhug.quirk"
+  File "share/fwupd/quirks.d/csr-aiaiai.quirk"
+  File "share/fwupd/quirks.d/dell-dock.quirk"
+  File "share/fwupd/quirks.d/dfu.quirk"
+  File "share/fwupd/quirks.d/ebitdo.quirk"
+  File "share/fwupd/quirks.d/fastboot.quirk"
+  File "share/fwupd/quirks.d/jabra.quirk"
+  File "share/fwupd/quirks.d/nitrokey.quirk"
+  File "share/fwupd/quirks.d/rts54hid.quirk"
+  File "share/fwupd/quirks.d/rts54hub.quirk"
+  File "share/fwupd/quirks.d/solokey.quirk"
+  File "share/fwupd/quirks.d/steelseries.quirk"
+  File "share/fwupd/quirks.d/synaptics-cxaudio.quirk"
+  File "share/fwupd/quirks.d/synaptics-prometheus.quirk"
+  File "share/fwupd/quirks.d/vli-pd.quirk"
+  File "share/fwupd/quirks.d/vli-usbhub-lenovo.quirk"
+  File "share/fwupd/quirks.d/vli-usbhub.quirk"
+  File "share/fwupd/quirks.d/wacom-usb.quirk"
+
+  ReadEnvStr $0 COMSPEC
+  SetOutPath "$INSTDIR"
+SectionEnd
+
+Section "Start Menu Shortcuts"
+  SetOutPath "$INSTDIR\bin"
+  CreateDirectory "$SMPROGRAMS\fwupd"
+  CreateShortCut "$SMPROGRAMS\fwupd\Uninstall fwupd.lnk" "$INSTDIR\Uninstall fwupd.exe" "" "$INSTDIR\Uninstall fwupd.exe" 0
+  SetOutPath "$INSTDIR"
+  CreateShortCut "$SMPROGRAMS\fwupd\fwupdtool.lnk" "$INSTDIR\bin\fwupdtool.exe" "" "$INSTDIR\bin\fwupdtool.exe" 0
```

Comment:
I don't think there is value in this being a shortcut.  It needs command line arguments, it shouldn't show up in the start menu.

Reply:
Agreed, fixed.